BetSystem/BusinnessLogic/SportEventService.cs

```diff
@@ -62,5 +67,98 @@ public void DeleteById(int id)
             _dbContext.Events.Remove(sportEvent);
             _dbContext.SaveChanges();
         }
+
+        public void AddTeamToSportEvent(int id, IdRequestDto idRequestDto)
+        {
+            var team = _dbContext.Teams.FirstOrDefault(x => x.Id == idRequestDto.Id);
+            _dbContext
+                .Events.FirstOrDefault(x => x.Id == id)
+                .Teams.Add(team);
+            _dbContext.SaveChanges();
+        }
+
+        public List<TeamDto> GetTeamsFromSportEvent(int id)
+        {
+            return _dbContext.Events
```

Comment:
Can we go from Teams.Where(x => x.Event.Id == id)? Or we don't have such relation?

Reply:
Don't have such relation.